README.md

```diff
@@ -95,6 +95,43 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## FAQ
+
```

Comment:
Add one more FAQ:

How to upgrade the existing deployment to use the latest model configuration? 

When using hosted public models, a user can delete the existing inference workload (Deployment of StatefulSet) manually, and the workspace controller will create a new one with the latest preset configuration (e.g., the image version) defined in the current release.  For private models, it is recommended to create a new workspace with a new image version in the Spec.

Reply:
Added